player/cPlayer.cpp

```diff
@@ -365,6 +365,12 @@ bool cPlayer::hasRadarAndEnoughPower() const {
     return getAmountOfStructuresForType(RADAR) > 0 && bEnoughPower();
 }
 
+std::string cPlayer::asString() const {
+    char msg[512];
+    sprintf(msg, "Player [id=%d, human=%b, sidebar=%d]", this->id, this->m_Human, this->sidebar);
```

Comment:
does it make sense to render `this->sidebar` as `%d` ?

Reply:
No! That's actually a bug. I'll fix it, as the code is being touched in this PR.